partner_duplicate_mgmt/tests/test_res_partner_duplicate.py

```diff
@@ -129,51 +122,43 @@ def test_05_char_field_merge_line_created_correctly(self):
             lambda l: l.duplicate_field_id == self.duplicate_email)
 
         self.assertTrue(merge_line)
-        self.assertEqual(merge_line.partner_1_value, 'partners@localhost')
-        self.assertEqual(merge_line.partner_2_value, 'contact_123@localhost')
+        self.assertEqual(merge_line.partner_1_value, 'contact_123@localhost')
+        self.assertEqual(merge_line.partner_2_value, 'partners@localhost')
 
     def test_06_many2one_field_merge_line_created_correctly(self):
         merge_line = self.contact_merge_lines.filtered(
             lambda l: l.duplicate_field_id == self.duplicate_state)
 
         self.assertTrue(merge_line)
-        self.assertEqual(merge_line.partner_1_value, 'Quebec')
-        self.assertEqual(merge_line.partner_2_value, 'Ontario')
-
-    def test_07_merge_partners_update_char_field_partner_2_preserved(self):
-        self.contact_dup.write({'partner_preserved_id': self.contact_1.id})
-
-        self.contact_merge_lines.write({'partner_1_selected': True})
-        self.contact_merge_lines.filtered(
-            lambda so: so.duplicate_field_id != self.duplicate_email
-        ).write({'partner_2_selected': True})
-
-        self.contact_dup.merge_partners()
-
-        self.assertEqual(self.contact_dup.state, 'merged')
-        self.assertEqual(self.contact_1.email, 'partners@localhost')
+        self.assertEqual(merge_line.partner_1_value, 'Ontario')
+        self.assertEqual(merge_line.partner_2_value, 'Quebec')
 
-    def test_08_merge_partners_update_char_field(self):
+    def test_07_merge_partners_update_char_field(self):
```

Comment:
@yasmineelmrini note à l'avenir, je préfère que tu laisses un trou dans la numérotation plutôt que de renommer toutes les fonctions.

Reply:
d'accord, c'est noté